db/migrate/20220310153625_create_exports.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.references :notebook, null: false, foreign_key: true
       t.references :user, null: false, foreign_key: true
       t.boolean :ready
+      t.boolean :failed
```

Comment:
This should be done in a new migration like `rails g migration add_failed_to_exports failed:boolean`.